src/Bitcoin.ts

```diff
@@ -0,0 +1,31 @@
+import * as bitcore from 'bitcore-lib'
+
+import { TransactionPoetTimestamp } from 'Interfaces'
+
+export const PREFIX_POET = Buffer.from('POET')
+export const PREFIX_BARD = Buffer.from('BARD')
+
+export function getPoetTimestamp(tx: bitcore.Transaction): TransactionPoetTimestamp {
```

Comment:
Move to /Helpers?

Reply:
I was actually thinking the BatchReader because its only used there? but Helpers could work too, not sure what makes most sense to you?

Reply:
Moved to helpers as that works good

Reply:
Yeah that's probably better.

Reply:
Oops, didn't see your last message. "Yeah that's probably better." was meant for "I was actually thinking the BlockchainReader because its only used there? but Helpers could work too, not sure what makes most sense to you?"

Either way is fine though